### ISSUE TITLE
Recommend to move this repo to the main library(jupyterlite/jupyterlite)

### ISSUE THREAD

**Issue by yiliang114** [jupyterlite/demo#125]
### Problem

https://jupyterlite.github.io/demo/lab/index.html 

The release of the sample site is maintained separately from the main library, resulting in untimely updates. For example, the main library has been updated to 0.2, but the demo site is still 0.1.

### Proposed Solution

Migrate actions to the main library(jupyterlite/jupyterlite)

**Comment by jtpio** [comment]
The demo lives in its own repo so it can be a GitHub template, which makes it very convenient to create a JupyterLite website in just a couple of clicks:

![image](https://github.com/jupyterlite/demo/assets/591645/54f65f48-dfb8-4264-badc-8e5e3a966cb0)

Sure `0.2.0a0` is out, but there are still a couple of remaining issues before we can make it the default: https://github.com/jupyterlite/jupyterlite/issues/1141

**Comment by jtpio** [comment]
If you want we can open a draft PR to update to `0.2.x`, for those who want to test the latest pre-releases.

**Comment by jtpio** [comment]
> If you want we can open a draft PR to update to `0.2.x`, for those who want to test the latest pre-releases.

Opened https://github.com/jupyterlite/demo/pull/126

**Comment by yiliang114** [comment]
> The demo lives in its own repo so it can be a GitHub template, which makes it very convenient to create a JupyterLite website in just a couple of clicks:
> 
> ![image](https://user-images.githubusercontent.com/591645/267922570-54f65f48-dfb8-4264-badc-8e5e3a966cb0.png)
> 
> Sure `0.2.0a0` is out, but there are still a couple of remaining issues before we can make it the default: [jupyterlite/jupyterlite#1141](https://github.com/jupyterlite/jupyterlite/issues/1141)

Thanks for such a quick reply. After listening to your description, I found that it is indeed faster to quickly deploy your own sample site.

**Comment by jtpio** [comment]
Sure, and thanks for testing the `0.2.0` pre-releases!

If you find any issue with `0.2.0a0` (and the next releases), please report in the JupyterLite repo, that would be really helpful :pray:  https://github.com/jupyterlite/jupyterlite

**Comment by jtpio** [comment]
Closing as answered. Don't hesitate to open a new issue if needed, thanks!
